book/eha.qmd

```diff
@@ -38,37 +40,50 @@ The *recurrent events setting* is as a special case of the multi-state setting b
+A common approach to competing-risks analysis is known as *cause-specific hazards*, where $h_{e}$ is defined as the hazard for the transition from initial state $0$ to state $e =1,\ldots,k$:
 
 $$
-h_{0e}(\tau) = \lim_{\Delta \tau \to 0} \frac{P(\tau \leq Y \leq \tau + \Delta \tau, E = e\ |\ Y \geq \tau)}{\Delta \tau}, \; e = 1, \dots, k.
+h_{e}(\tau) = \lim_{\Delta \tau \to 0} \frac{P(\tau \leq Y \leq \tau + \Delta \tau, E = e\ |\ Y \geq \tau)}{\Delta \tau}, \; e = 1, \dots, k.
 $$
 
-As competing events are mutually exclusive at any time $\tau$, it is useful to further define the:
+As competing events are mutually exclusive at any time $\tau$, it is useful to further define the *all-cause hazard* which is the hazard of any event occuring:
 
-- *all-cause hazard*, $h(\tau) = \sum_{e = 1}^{k}h_{0e}(\tau)$: which is the...
-- *all-cause cumulative hazard*: $H(\tau) = \int_{0}^\tau h(u)\ du = \sum_{e=1}^{k}\int_{0}^\tau h_{0e}(u)\ du = \sum_{e=1}^k H_{0e}(\tau)$, and
-- *all-cause survival probability*: $S(\tau) = P(Y \geq \tau) = \exp(-H(\tau))$
+$$
+h(\tau) = \sum_{e = 1}^{k}h_{e}(\tau)
+$$
+
+the *all-cause cumulative hazard*
+
+$$
+H(\tau) = \int_{0}^\tau h(u)\ du = \sum_{e=1}^{k}\int_{0}^\tau h_{e}(u)\ du = \sum_{e=1}^k H_{e}(\tau)
+$$
+
+and the *all-cause survival probability*
 
-Note that whilst the all-cause survival probability is defined in the same way as the standard survival probability, the interpretation is that *none* of the events occurred before $\tau$.
-The all-cause survival probability is also not directly estimated but instead calculated from the cause specific hazards.
+$$
+S(\tau) = P(Y > \tau) = \exp(-H(\tau))
+$$
 
-Another important distribution function in competing risks analysis is the *Cumulative Incidence Function* (CIF), which denotes the probability of experiencing an event $E$ before time $\tau$
+While notationally identical to the survival probability in the single-event case, the all-cause survival probability in the competing risks setting implies that *none* of the events occurred before $\tau$.
+It is often not estimated directly but calculated from the cause specific hazards instead.
+
+Another important quantity in competing risks analysis is the *Cumulative Incidence Function* (CIF), which denotes the probability of experiencing an event $e$ before time $\tau$
 
 $$
-F_{e}(\tau) = P(Y \leq \tau, E = e) = \int_{0}^\tau h_{0e}(u)S(u)\ du.
+F_{e}(\tau) = P(Y \leq \tau, E = e) = \int_{0}^\tau S(u)h_{e}(u)\ \mathrm{d}u.
 $$
-Intuitively the CIF is the probability of not experiencing any event, $S(u)$, multiplied with the hazard of experiencing an event of type $e$, $h_{oe}(u)$.
 
+Intuitively the contributions in the integral consist of the probability of not experiencing any event before time $u$, $S(u)$, multiplied with the hazard of experiencing an event of type $e$, $h_{oe}(u)$ immediately at time $u$.
```

Comment:
I don't think it's intuitive. I'm always confused about the CIF, especially because it's the same notation as the CDF. I think some more text here would be useful

Reply:
I think it's best to have examples here, with plots, etc.